web-console/4.0-designs/help-menu/help-menu.md

```diff
@@ -0,0 +1,29 @@
+# Help Menu
+
+The help menu will be accessible at all times from the primary masthead. Users can access it by clicking the help menu dropdown (fa-question-circle-o).
+
+**Menu Dropdown**
+![menu](img/menu.png)
+
+* The menu dropdown will include links to Documentation, the Quick Start Guide, the Command Line Tools page, and the About modal
+  * Documentation will redirect users to an external link
+  * Quick Start Guide will take users to the Quick Start Guide landing page
+  * Command Line Tools will redirect users to the CLI page
+  * About will open the About modal
+
+
+
+**Quick Start Guide**
+![Quick Start Guide](img/quick-start-guide.png)
+
+* The Quick Start Guide landing page will have the quick start guide info as well as information regarding documentation and additional support links
+
+**CLI page**
+![CLI](img/CLI-page.png)
+
+* The CLI page will have information about the oc client and command line tools
+
+**About**
+![About](img/about-modal.png)
```

Comment:
@bmignano let's check with Kyle today since we never had the About modal implemented in OS. We should get a high fidelity version of this modal created if possible.